python/aie/compiler/aiecc/main.py

```diff
@@ -195,7 +196,7 @@ async def process_core(self, core):
         await self.do_call(task, ['aie-translate', self.file_with_addresses, '--aie-generate-ldscript', '--tilecol=%d' % corecol, '--tilerow=%d' % corerow, '-o', file_core_ldscript])
       if(not self.opts.unified):
         file_core_llvmir = self.tmpcorefile(core, "ll")
-        await self.do_call(task, ['aie-translate', '--opaque-pointers=0', '--mlir-to-llvmir', file_opt_core, '-o', file_core_llvmir])
+        await self.do_call(task, ['aie-translate', '--opaque-pointers=1', '--mlir-to-llvmir', file_opt_core, '-o', file_core_llvmir])
```

Comment:
`aie-translate` complained `--opaque-pointers=1` was an invalid argument. I just removed it get it to work.